scrapy_crawlera/middleware.py

```diff
@@ -3,6 +3,11 @@
 import warnings
 from collections import defaultdict
 
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
```

Comment:
`urlparse` is already imported from `six` in line below

Reply:
Thanks, how could I have missed that :)